CHAOS-4162 ChaosHub changes (#5326)
harness/developer-hub: docs/chaos-engineering/configure-chaos-experiments/chaos-hubs/manage-hub.md

---

> title: Manage custom chaoshub

Reviewer: Following the convention used in other places
```suggestion
title: Manage custom ChaosHub
```

sidebar_position: 3

> description: Save and manage experiments in custom chaoshub

Reviewer: Following the convention used in other places
```suggestion
description: Save and manage experiments in custom ChaosHub
```

---

import Launch from '../shared/launch-experiment-from-hub.md'

This section describes how to save and manage experiments in your custom chaoshub(s), and how to sync your ChaosHub GitHub repo with Harness. Go to [Add a custom ChaosHub](/docs/chaos-engineering/configure-chaos-experiments/chaos-hubs/add-chaos-hub) for setup instructions.

## View ChaosHubs

> The experiments you see in various ChaosHub are actually templates from which you can launch the experiments after specifying some details. 

Reviewer: ```suggestion
The experiments you see in various ChaosHubs are templates from which you can launch the experiments after specifying some details. 
```

To view the default and custom ChaosHub:

1. In Harness, navigate to **Chaos > ChaosHubs**.

> 	This page lists the default Enterprise ChaosHub, and any custom ChaosHub.

Reviewer: ```suggestion
	This page lists the default Enterprise ChaosHub and any custom ChaosHubs.
```
